test/io_tests.cpp

```diff
@@ -325,7 +351,9 @@ TEST(io_tests, can_encode_large_files) {
 			(factory::record_item::GPSTIME)
 			(factory::record_item::RGB12);
 
-		io::writer::file f("/tmp/autzen_trim.laz", schema,
+		char fname[L_tmpnam];
```

Comment:
There wasn't another method. I was just hard coded to `/tmp/autzen_trim.laz`. I guess I'm confused here.